etc/containerpilot.json

```diff
@@ -49,9 +49,7 @@
     {{ if and .CONSUL_AGENT .ACME_DOMAIN }},{{ end }}
     {{ if .ACME_DOMAIN }}
     {
-      "command": ["/usr/local/bin/consul-template",
-                  "-config", "/etc/acme/watch.hcl",
-                  "-consul", "{{ if .CONSUL_AGENT }}localhost{{ else }}{{ if .CONSUL }}{{ .CONSUL }}{{ else }}consul{{ end }}{{ end }}:8500"],
+      "command": ["acme", "watch"],
```

Comment:
Why not just use `/usr/local/bin/consul-template -config /etc/acme/watch.hcl -consul $CONSUL_HOST:8500` directly here?